static/templates/stream_settings/stream_types.hbs

```diff
@@ -1,3 +1,10 @@
+{{#if ask_to_announce_stream}}
+    <div id="announce-new-stream">
+        {{> announce_stream }}
+    </div>
+{{/if}}
```

Comment:
Probably it is better to move this code to `stream_creation_form.hbs` now. And then we can remove the `ask_to_announce_stream` condition as well.